datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt

```diff
@@ -102,6 +102,7 @@ object QuestionnaireItemDatePickerViewHolderFactory : QuestionnaireItemViewHolde
                         context.supportFragmentManager,
                         DatePickerFragment.TAG
                     )
+                    textInputLayout.clearFocus()
```

Comment:
please add the following comment (or something similar) and add the same to the other two placess

```suggestion
                    // Clear focus so that the user can refocus to open the dialog
                    textInputLayout.clearFocus()
```